### ISSUE TITLE
Need help in my code

### ISSUE THREAD

**Issue by Ooip83** [Ooip83/How-to-control-multiple-servo-motors-at-once-#1]
Below are my code...
#include <Servo.h>
Servo myservo1;
Servo myservo2;
Servo myservo3;
const int button = 3;
int previousState = LOW;
void setup() {
  myservo1.attach(5);
  myservo1.attach(6);
  myservo1.attach(7);
  pinMode(button, INPUT);
  Serial.begin(9600);

}

void loop() {
  int currentState = digitalRead(button);
  Serial.println(currentState);
  delay(500);
  if (currentState == HIGH && previousState == LOW){
  myservo1.write(90);
  myservo2.write(90);
  myservo3.write(90);
  delay(1000);
  previousState = HIGH;
  }

  if (currentState == HIGH && previousState == HIGH){
  myservo1.write(0);
  myservo2.write(0);
  myservo3.write(0);
  delay(1000);
  previousState = LOW;
  }
  

}
..................................................................................................................................................................................................................
Basically, i want to make the three servos to move together, but not one by one.
Sorry for my bad english as english is not my first language...

**Comment by Ooip83** [comment]
Can I just connect all the signal wire(orange colour) to one digitalpin(e.g. pin5) to unify them, so that all three servos move together? I personally think it won't work.
